projects/opencv/Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 
-RUN apt-get update && apt-get install -y build-essential cmake pkg-config
+RUN apt-get update && apt-get install -y build-essential cmake libgif-dev pkg-config
```

Comment:
Do we really need `libgif-dev`? I thought we have built-in format support: https://github.com/opencv/opencv/pull/25691

Reply:
Indeed, I simplified the commits. Thx !